index.html

```diff
@@ -0,0 +1,36 @@
+      
+<h2 >"Dr.Norman Borlaung Tribute"</h2>
+<div class="top-section">
+<p><span>Norman Ernest Borlaug,</span> (born March 25, 1914, near Saude, Iowa, U.S.—died September 12, 2009, Dallas, Texas), American agricultural scientist, plant pathologist, and winner of the Nobel Prize for Peace in 1970. Known as the “Father of the Green Revolution,” Borlaug helped lay
+the groundwork for agricultural technological advances that alleviated world hunger.
+</p>
+/div>     
+<div id="img-div">
+      
+<div id="img-caption">The Indian Vectory leader</div>
+      
+</div>
+<div class="middle-section"><p>
+To finance his studies, Borlaug put his education on hold periodically to earn some income, as he did in 1935 as a leader in the Civilian Conservation Corps, working with the unemployed on Federal projects. Many of the people who worked for him were starving. He later recalled, "I saw how food changed them ... All of this left scars on me".[17] From 1935 to 1938, before and after receiving his Bachelor of Science in forestry in 1937, Borlaug worked for the United States Forest Service at stations in Massachusetts and Idaho. He spent one summer in the middle fork of Idaho's Salmon River, the most isolated piece of wilderness in the nation at that time.[17]
+
+In the last months of his undergraduate education, Borlaug attended a Sigma Xi lecture by Elvin Charles Stakman, a professor and soon-to-be head of the plant pathology group at the University of Minnesota. The event was a pivot for Borlaug's future. Stakman, in his speech entitled "These Shifty Little Enemies that Destroy our Food Crops", discussed the manifestation of the plant disease rust, a parasitic fungus that feeds on phytonutrients in wheat, oats, and barley crops. He had discovered that special plant breeding methods produced plants resistant to rust. His research greatly interested Borlaug, and when Borlaug's job at the Forest Service was eliminated because of budget cuts, he asked Stakman if he should go into forest pathology. Stakman advised him to focus on plant pathology instead.[16] He subsequently enrolled at the university to study plant pathology under Stakman. Borlaug earned a Master of Science degree in 1940, and a Ph.D. in plant pathology and genetics in 1942. 
+</p></div>
+      
+<p id="tribute-info">The Indian Vectory leader</p>
+      
+<a id="tribute-link" href="https://www.microverse.org/" target="_blank">More Details...</a>
+</main>
+</body>
+</html>
```

Comment:
Please I think the webpage should be split into sections using the section tags instead of the div tags for easy readability and also for the code to be clean.

Reply:
- This would have been a great review but I think with your keen eye, you can look more closely enough to have a more clear suggestion to the student because right now, this comment is not clear enough. The student might wonder if you want him/her to replace `div`s with `section` tags, or get confused with what you meant by the statement `"to split the webpage into sections`"
- [ ] Let's ensure that our suggestions are clear and understandable without any ambiguous meaning so that students don't get blocked or caught in a review loop. To this effect, I think restructuring this in a clear manner would help.

For Example:

  - This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

  - This is a vague comment: The README is not descriptive, why don't you make it better?